.github/workflows/update-bootstrap.yml

```
@@ -0,0 +1,44 @@
+name: Update Bootstrap nodes
+
+on:
+  release:
+    types: [released]
+
+jobs:
+  update-bootstrap-nodes:
+    name: Update Bootstrap nodes
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        environment: [bootstrap-devqa, bootstrap-tnet-mainnet]
```

Can add the `limit` here:
```suggestion
      matrix:
        include:
          - environment: bootstrap-tnet-mainnet
            limit: tnet,mainnet
          - environmnt: bootstrap-devqa
            limit: devqa
```